core/app/[locale]/(default)/product/[slug]/page.tsx

```diff
@@ -164,6 +165,10 @@ export default async function Product(props: Props) {
         quantityLabel={t('ProductDetails.quantity')}
       />
 
+      <Stream fallback={null} value={productPromise}>
+        {(product) => <Reviews productId={product.entityId} />}
+      </Stream>
```

Comment:
I wonder if I should use the `ReviewSkeleton` here, or if we don't really care since this should live below the fold. 🤔

Reply:
Yeah, should definitely use a skeleton. Feel free to decouple from this PR if you want though

Reply:
Wait, aren't we already handling streaming inside the `Reviews` component? Why do we need to use `Stream` here?

Reply:
🤦 You're right! Updated!